content/docs/csidriver/installation/helm/unity.md

```diff
@@ -233,60 +233,60 @@ Procedure
     A successful installation must display messages that look similar to the following samples:
     ```
     ------------------------------------------------------
-    > Installing CSI Driver: csi-unity on 1.20
+    > Installing CSI Driver: csi-unity on 1.22
     ------------------------------------------------------
     ------------------------------------------------------
     > Checking to see if CSI Driver is already installed
     ------------------------------------------------------
     ------------------------------------------------------
     > Verifying Kubernetes and driver configuration
     ------------------------------------------------------
-    |- Kubernetes Version: 1.20
+    |- Kubernetes Version: 1.22
     |
     |- Driver: csi-unity                                                
     |
-    |- Verifying Kubernetes versions                                    
-      |
-      |--> Verifying minimum Kubernetes version                         Success
-      |
-      |--> Verifying maximum Kubernetes version                         Success
+    |- Verifying Kubernetes version                                     
     |
-    |- Verifying that required namespaces have been created             Success
+    |--> Verifying minimum Kubernetes version                         Success
     |
-    |- Verifying that required secrets have been created                Success
+    |--> Verifying maximum Kubernetes version                         Success
     |
-    |- Verifying that required secrets have been created                Success
+    |- Verifying that required namespaces have been created           Success
+    |
+    |- Verifying that required secrets have been created              Success
```

Comment:
Same as line 258. Is this how it should be?

Reply:
@karthikk92 : why 256 and 258 lines are repeated is it a bug?

Reply:
@shanmydell  same function is getting called for both empty and unity-creds as well 
![image](https://user-images.githubusercontent.com/92289639/150909413-7eb7e48f-8104-4091-8d3c-a473b1c01ed8.png)

Reply:
![image](https://user-images.githubusercontent.com/92289639/150909537-83878df1-0b1f-4da2-80fb-29c18300aaa5.png)
is this change looks ok?

Reply:
@karthikk92 : These changes need to be ported to all the drivers, Please track via a CSM issue. Message can be in simple terms like below.
Verifying that required secrets for creds have been created 
Verifying that required secrets for certs have been created

Reply:
@shanmydell @AronAtDell   taken care now